### PR TITLE
Add new method for language code

### LINKED ISSUE #612: Kiwix-serve does not display properly the Ladakhi ISO 639-2 code
If I load https://mirror.download.kiwix.org/zim/videos/dse-ladak_hi_class-3rd_2021-08.zim with Kiwix-serve, I get this on the Welcome Page:
image

Ladakhi does not seem to have the proper ISO 639-2 language code. Actually the Ladakhi language does not have one AFAIK. So I wonder what is wrong here. Probably the ISO 639-3 language code should be displayed here if there is not ISO 639-2 variant. 

See more info about Ladakhi language at https://en.wikipedia.org/wiki/Ladakhi_language

### PR DESCRIPTION
**Pull request by juuz0**
Fixes #612

### CODE REVIEW

File: static/skin/index.js [hunk 1]
```diff
@@ -114,12 +114,18 @@
...
             </div>
             <div class="book__description" title="${description}">${description}</div>
-            <div class="book__languageTag" ${languageAttr}>${language.substr(0, 2).toUpperCase()}</div>
+            <div class="book__languageTag" ${languageAttr}>${getLanguageCode(language)}</div>
             <div class="book__tags"><div class="book__tags--wrapper">${tagHtml}</div></div>
             </div></div></a>`;
         return divTag;
     }
 
+    function getLanguageCode(langCode3Letter) {
```
Comment (kelson42) [inline static/skin/index.js:123]: Better rename to `getLanguageCodeToDisplay()`

File: static/skin/iso6391To3.js [hunk 2]
```diff
@@ -1,85 +1,134 @@
 // eslint-disable-next-line no-unused-vars
 const iso6391To3 = {
```
Comment (kelson42) [inline static/skin/iso6391To3.js:2]: Please put the source of this list, if you have choice, please try to find a source of reference (unicode consortium or reference implementation).

File: static/skin/index.js [hunk 3]
```diff
@@ -114,12 +114,18 @@
                 ${downloadLink ? `<div class="book__download"><span data-link="${downloadLink}">Download ${humanFriendlyZimSize ? ` - ${humanFriendlyZimSize}</span></div>`: ''}` : ''}
             </div>
             <div class="book__description" title="${description}">${description}</div>
-            <div class="book__languageTag" ${languageAttr}>${language.substr(0, 2).toUpperCase()}</div>
+            <div class="book__languageTag" ${languageAttr}>${getLanguageCode(language)}</div>
```
Comment (kelson42) [inline static/skin/index.js:117]: I see nowhere `iso6391To3.js` included properly in the HTML. Are you sure this works?

**Review by kelson42 — CHANGES_REQUESTED** [review]
Principle looks good, only few details to fix.

File: static/skin/index.js (hunk 3 above)
Comment (juuz0) [reply static/skin/index.js:117]: Included in `index.html` line 42
Used previously in `index.js` line 412

File: static/skin/iso6391To3.js (hunk 2 above)
Comment (juuz0) [reply static/skin/iso6391To3.js:3]: Used this https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes

**Review by kelson42 — APPROVED** [review]
LGTM

**Comment by kelson42** [comment]
@juuz0 I have rebased and reorganised the commit. Will merge. Thx!

**Pull request merged**